app/controllers/items_controller.rb

```diff
@@ -5,6 +5,8 @@ def index
       @merchant = Merchant.find(params[:merchant_id])
       @items = @merchant.items
     else
+      @top_five_order_items = ItemOrder.top_five
+      @bottom_five_order_items = ItemOrder.bottom_five
```

Comment:
I call these methods in the index view. Methods are located in Item Order model.